services/outputhost/cgcache.go

```diff
@@ -347,6 +349,26 @@ func (cgCache *consumerGroupCache) loadExtentCache(destType shared.DestinationTy
 	return
 }
 
+// unloadExtentCache unloads an extent from the extentCache
+func (cgCache *consumerGroupCache) unloadExtentCache(extUUID string) (ackMgrUnloadID uint32) {
+	if extCache, ok := cgCache.extentCache[extUUID]; ok {
+		extCache.logger.Info("removing extent")
+		ackMgrUnloadID = uint32(extCache.ackMgr.ackMgrID)
+		cgCache.cgMetrics.Decrement(load.CGMetricNumOpenExtents)
+		cgCache.hostMetrics.Decrement(load.HostMetricNumOpenExtents)
+		if extCache.singleCGVisible {
+			dlqMerging := atomic.AddInt32(&cgCache.dlqMerging, -1)
+			if dlqMerging <= 0 { // Note: this should never be less than 0; writing it like this for diagnosis
+				cgCache.logger.Info("done merging DLQ extent(s)")
```

Comment:
This log line isn't even potentially bad, despite the comment. Any customer that has a problem with this will present with a story about how they did a DLQ merge, and now their health metrics show smart retry, which should be enough clues.